rclcpp/include/rclcpp/executor.hpp

```diff
@@ -174,6 +174,26 @@ class Executor
   virtual void
   spin_some(std::chrono::nanoseconds max_duration = std::chrono::nanoseconds(0));
 
+  /// Complete all available queued work without blocking.
+  /**
+   * This function can be overridden. The default implementation is suitable for a
+   * single-threaded model of execution.
+   * Adding subscriptions, timers, services, etc. with blocking callbacks will cause this function
+   * to block (which may have unintended consequences).
+   *
+   * To ensure all available queued work is executed, this function have to check if there is
```

Comment:
```suggestion
   * To ensure all available queued work is executed, this function has to check if there is
```

Reply:
Thanks! Fixed in https://github.com/ros2/rclcpp/pull/1156/commits/7d43e5b27adbf0cfe87ed6ebd4168d47b4ef4283.